u_common.go

```diff
@@ -74,49 +74,68 @@ const (
 	helloRandomizedNoALPN
 )
 
+type ClientHelloSpec struct {
+	ClientRandom       []byte
+	CipherSuites       []uint16       // nil => default
+	CompressionMethods []uint8        // nil => no compression
+	Extensions         []TLSExtension // nil => no extensions
+
+	// GreaseStyle: currently only Chrome(GREASE depends on ClientRandom)
```

Comment:
Does GREASE still depend on ClientRandom?

Reply:
Good point, forgot to update this.